ingest/expression_files/mtx.py

```diff
@@ -75,6 +82,26 @@ def check_valid(
             raise ValueError("; ".join(error_messages))
         return True
 
+    @staticmethod
+    def is_sorted(file_path: str):
+        """Checks if a file is sorted by gene index"""
+        file_size = os.path.getsize(file_path)
+        if file_size == 0:
+            raise ValueError(f"{file_path} is empty: " + str(file_size))
+        start_idx: int = MTXIngestor.get_line_no(file_path)
+        p1 = subprocess.Popen(
+            ["tail", "-n", f"+{start_idx}", f"{file_path}"], stdout=subprocess.PIPE
+        )
+        p2 = subprocess.run(
+            ["sort", "-c", "--stable", "-n", "-k", "1,1"],
```

Comment:
this needs a comment explaining what those options do

Reply:
125b965